include/picongpu/param/particle.param

```diff
@@ -117,57 +128,82 @@ namespace picongpu
                     particle[weighting_] *= 2.0_X;
                 }
             };
-
-            /** definition of a free particle manipulator: double weighting */
+            /** Definition of the free particle manipulator which doubles each weighting.*/
             using DoubleWeighting = generic::Free<DoubleWeightingFunctor>;
 
+
+            /** Define mask which randomly marks macro-particles used
+             *  by Radiation plugin to calculate far field radiation.
+             */
             struct RandomEnabledRadiationFunctor
             {
                 template<typename T_Rng, typename T_Particle>
                 DINLINE void operator()(T_Rng& rng, T_Particle& particle)
                 {
-                    // enable radiation for 10% of the particles
+                    // enable radiation for 10% of the macro-particles
                     particle[radiationMask_] = rng() < 0.1_X;
                 }
             };
-
-            /* definition of RandomEnableRadiation start */
+            /** Definition of manipulator that selects macro-particles for Radiation plugin. */
             using RandomEnabledRadiation
                 = generic::FreeRng<RandomEnabledRadiationFunctor, pmacc::random::distributions::Uniform<float_X>>;
 
-            /** changes the in-cell position of each particle of a species */
+            /** Definition of manipulator that changes the in-cell position of each particle of a species. */
             using RandomPosition = unary::RandomPosition;
 
         } // namespace manipulators
 
         namespace startPosition
         {
+            /** Define target number for marco-particles per cell
+             * to be used in Random start position functor.
+             */
             struct RandomParameter
             {
-                /** Count of particles per cell at initial state
+                /** Count of macro-particles per cell at initial state
                  *
                  *  unit: none */
                 static constexpr uint32_t numParticlesPerCell = TYPICAL_PARTICLES_PER_CELL;
             };
-            /** definition of random particle position */
+            /** Definition of start position functor that randomly distributes macro-particles within a cell. */
             using Random = RandomImpl<RandomParameter>;
 
-            /** definition of random particle position and weighting, same template parameter */
+
+            /** Definition of start position functor that randomly distributes macro-particles within a cell
+             * and randomly distributes weighting across macro-particles within a cell.
```

Comment:
not sure - should it be weighting**s** ?

Reply:
Done